plugins/pr-to-oc-tests/commands/generate-execution-steps.md

```
@@ -0,0 +1,375 @@
+---
+description: Generate step-by-step test execution procedure for any PR
+argument-hint: "<pr-url> [--env <cluster-type>]"
+---
+
+## Name
+pr-test-generator:generate-execution-steps
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)